platform/ios/src/MGLMapView.mm

```diff
@@ -240,6 +240,9 @@ @interface MGLMapView () <UIGestureRecognizerDelegate,
 @property (nonatomic) MGLUserLocation *userLocation;
 @property (nonatomic) NSMutableDictionary<NSString *, NSMutableArray<MGLAnnotationView *> *> *annotationViewReuseQueueByIdentifier;
 
+/// A Boolean value that determines whether the updating pitch will also affect the altitude.
+@property(nonatomic, getter=isCameraAltitudeAffectedByPitch) BOOL cameraAltitudeAffectedByPitch;
```

Comment:
A more natural name for the getter would be `cameraAltitudeIsAffectedByPitch` or, given https://github.com/mapbox/mapbox-gl-native/pull/12518#discussion_r207418088, `cameraAltitudeIsViewingDistance`.